lib/server/generate.js

```diff
@@ -20,30 +20,15 @@ function execute() {
   const glob = require('glob');
   const chalk = require('chalk');
   const Site = require('../core/Site.js');
+  const env = require('./env.js');
   const siteConfig = require(CWD + '/siteConfig.js');
   const translate = require('./translate.js');
-  const versionFallback = require('./versionFallback.js');
```

Comment:
I am assuming this is removed because it is unused.

Reply:
yes this one is not used.